test/references/retry_tests_report_1worker.xml

```diff
@@ -0,0 +1,328 @@
+		</error>
+	</testcase>
+	<testcase name="Timeout always" timestamp="2023-06-21T13:40:26.634" time="0.0" tests="1" skipped="0" failures="0" errors="1">
+		<properties>
+		<property name="dd_tags[perf.bytes]" value="0"></property>
+		<property name="dd_tags[perf.allocs]" value="0"></property>
+		<property name="dd_tags[perf.gctime]" value="0.0"></property>
+		<property name="dd_tags[perf.compile_time]" value="0.0"></property>
+		<property name="dd_tags[perf.recompile_time]" value="0.0"></property>
+		<property name="dd_tags[perf.eval_time]" value="0.0"></property>
+		</properties>
+		<error message="Error during test at test/testfiles/_retry_tests.jl:52">Captured Logs for test item "Timeout always" at test/testfiles/_retry_tests.jl:52 on worker 51271
+
+signal (15): Terminated: 15
+in expression starting at /Users/nickr/repos/ReTestItems.jl/test/testfiles/_retry_tests.jl:58
+__psynch_cvwait at /usr/lib/system/libsystem_kernel.dylib (unknown line)
+unknown function (ip: 0x0)
+kevent at /usr/lib/system/libsystem_kernel.dylib (unknown line)
+unknown function (ip: 0x0)
+Allocations: 3068784 (Pool: 3067673; Big: 1111); GC: 3
+Error in testset "Timeout always" on worker 51271:
+Error During Test at /Users/nickr/repos/ReTestItems.jl/test/testfiles/_retry_tests.jl:52
+  Got exception outside of a @test
+  Timed out after 5s evaluating test item "Timeout always" (run=3)
+		</error>
+	</testcase>
+	<testcase name="Timeout first, pass after" timestamp="2023-06-21T13:40:32.502" time="0.0" tests="1" skipped="0" failures="0" errors="1">
+		<properties>
+		<property name="dd_tags[perf.bytes]" value="0"></property>
+		<property name="dd_tags[perf.allocs]" value="0"></property>
+		<property name="dd_tags[perf.gctime]" value="0.0"></property>
+		<property name="dd_tags[perf.compile_time]" value="0.0"></property>
+		<property name="dd_tags[perf.recompile_time]" value="0.0"></property>
+		<property name="dd_tags[perf.eval_time]" value="0.0"></property>
+		</properties>
+		<error message="Error during test at test/testfiles/_retry_tests.jl:62">Captured Logs for test item "Timeout first, pass after" at test/testfiles/_retry_tests.jl:62 on worker 51478
+
+signal (15): Terminated: 15
+in expression starting at /Users/nickr/repos/ReTestItems.jl/test/testfiles/_retry_tests.jl:69
+__psynch_cvwait at /usr/lib/system/libsystem_kernel.dylib (unknown line)
+unknown function (ip: 0x0)
+kevent at /usr/lib/system/libsystem_kernel.dylib (unknown line)
+unknown function (ip: 0x0)
+Allocations: 3061069 (Pool: 3059958; Big: 1111); GC: 3
+Error in testset "Timeout first, pass after" on worker 51478:
+Error During Test at /Users/nickr/repos/ReTestItems.jl/test/testfiles/_retry_tests.jl:62
+  Got exception outside of a @test
+  Timed out after 5s evaluating test item "Timeout first, pass after" (run=1)
+		</error>
+	</testcase>
+	<testcase name="Timeout first, pass after" timestamp="2023-06-21T13:40:33.616" time="0.062" tests="1" skipped="0" failures="0" errors="0">
```

Comment:
there should be 2 `<testcase name="Timeout first, pass after"` (one timeout, then one pass)